src/app/interactions/query-builder/query-builder-sql-sample/query-builder-sql-sample.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="wrapper">
+    <div class="container">
+        <igx-query-builder #queryBuilder
```

Comment:
How about adding the `output-area` class to QB:
```suggestion
        <igx-query-builder #queryBuilder class="ouput-area"
```
This way we get the same border as the one around the SQL query:

![image](https://github.com/user-attachments/assets/73a37155-19fc-4b8c-b356-8e840993e597)